source/diagrams/manage-diagrams/clone-diagrams.txt

```diff
@@ -1,13 +1,68 @@
 .. _rm-clone-diagrams:
 
-==============
-Clone Diagrams
-==============
+===============
+Clone a Diagram
+===============
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Clone diagrams.
\ No newline at end of file
+You can clone existing diagrams on the :guilabel:`Mapping` screen of
+Relational Migrator. 
+
+About this Task
+---------------
+
+- You can clone one or more diagrams to create new diagrams that contain
+  the same entities as existing diagrams. 
+- Once you clone a diagram, you can edit, :ref:`rename
+  <rm-rename-diagrams>`, and :ref:`delete <rm-delete-diagrams>` your new
+  diagram. 
+
+
+Steps
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Navigate to the diagram you want to clone
+
+      Click the name of the diagram in the Diagram Toolbar to display the
+      diagram. 
+
+      .. image:: /img/diagrams/diagram-toolbar.png
+         :alt: Diagram toolbar    
+
+   .. step:: Open the :guilabel:`Diagram Properties` pane
+
+      Click the :icon-fa5:`chevron-circle-right` arrow button to open the
+      :guilabel:`Diagram Properties` pane. 
+
+      .. image:: /img/diagrams/diagram_properties.png
+         :alt: Diagram Properties pane
+
+      The :guilabel:`Diagram Properties` pane displays the active diagram name and
+      notes. 
+
+   .. step:: Click the :icon-fa5:`ellipsis-h` button in the :guilabel:`Diagram Properties` pane
+
+      A menu opens. 
```

Comment:
Suggest putting this line after the image to stay consistent.

Reply:
Done!